dali/pipeline/executor/workspace_policy.h

```diff
@@ -171,21 +171,21 @@ void SetupInputOutput(op_type_to_workspace_t<op_type> &ws, const OpGraph &graph,
 
 template <OpType op_type>
 inline void SetupThreadPool(op_type_to_workspace_t<op_type> &ws, const OpGraph &, const OpNode &,
-                            ThreadPool *, const QueueIdxs) {
+                            ThreadPool *, const QueueIdxs &) {
   /* No-op if we are not CPU */
 }
 
 template <>
 inline void SetupThreadPool<OpType::CPU>(op_type_to_workspace_t<OpType::CPU> &ws, const OpGraph &,
-                                         const OpNode &, ThreadPool *thread_pool, const QueueIdxs) {
+                                         const OpNode &, ThreadPool *thread_pool, const QueueIdxs &) {
```

Comment:
```suggestion
                                         const OpNode &,
                                         ThreadPool *thread_pool, const QueueIdxs &) {
```
Line length limit of 100 exceeded.